docs/en/helm/known-issues.md

```diff
@@ -0,0 +1,55 @@
+# Limitations and known issues
+
+Here are the highlights for using Helm Chart as schematic module. 
+
+## Only one main workload in the Chart
+
+The Chart must have exactly one workload considered as the main workload, e.g.,
+Deployment, CloneSet, etc. 
+In our context, `main workload` means the workload that will be tracked by
+KubeVela controllers, applied with Traits and added into Scopes. 
+
+To tell KubeVela which one is the main workload, two tasks are required:
+
+#### 1. Declare main workload's resource definition
+
+The field, `.spec.definitionRef` , in `WorkloadDefinition` is used to record the
+resource definition of the main workload. 
+The name should be in the format: `<resource>.<group>`. 
+
+```yaml
+apiVersion: core.oam.dev/v1alpha2
+kind: WorkloadDefinition
+...
+spec:
+  definitionRef:
+    name: deployments.apps
+    version: v1
+```
+```yaml
+...
+spec:
+  definitionRef:
+    name: clonesets.apps.kruise.io
+    version: v1alpha1
+```
+
+#### 2. Qualified full name of the main workload
+
+The name of the main workload should be templated with `qualifiied full name` by
```

Comment:
```suggestion
The name of the main workload should be templated with `qualified full name` by
```

Please add a link here about what is the `qualifiied full name`

Reply:
ok